internal/cli/cli.go

```diff
@@ -38,3 +41,17 @@ func read(path string) ([]byte, error) {
 	}
 	return os.ReadFile(path)
 }
+
+func defaultProject(ctx context.Context) (string, error) {
+	p := project.FromApplicationDefault(ctx)
+	if p != "" {
+		return p, nil
+	}
+
+	p = project.FromEnv()
+	if p != "" {
+		return p, nil
+	}
+
+	return "", fmt.Errorf("missing project")
```

Comment:
I don't think missing project is an error but a valid state. We can return can bool instead of the error.